README.md

```diff
@@ -378,3 +378,4 @@ The following issues are currently being explored and input from the community i
 - Tony Kovanen ([@tonykovanen](https://twitter.com/tonykovanen)) – ▲ZEIT
 - Guillermo Rauch ([@rauchg](https://twitter.com/rauchg)) – ▲ZEIT
 - Dan Zajdband ([@impronunciable](https://twitter.com/impronunciable)) – Knight-Mozilla / Coral Project
+
```

Comment:
Could you remove it since it's not related to license ?
And please squash all commits to one.